.gitignore

```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .yarn/install-state.gz
+env/
```

Comment:
Why do we need an `/env` directory? We have the `.env` file for the env variables, but the `/env` directory is not something that we are currently using. Can you elaborate on the need of this?